bin/gwc/db1link.ml

```diff
@@ -1301,6 +1301,8 @@ let link next_family_fun bdir =
       output_wizard_notes bdir gen.g_wiznotes;
       Mutil.remove_dir tmp_dir ;
       output_command_line bdir;
+      Mutil.rm (Filename.concat bdir "synchro_patches");
```

Comment:
I think these ones should be added directly here : https://github.com/geneweb/geneweb/blob/master/lib/gwdb-legacy/outbase.ml#L343